openmetadata-ui/src/main/resources/ui/src/components/TeamDetails/TeamDetailsV1.tsx

```diff
@@ -197,16 +199,19 @@ const TeamDetailsV1 = ({
           </Link>
         ),
       },
-      {
-        title: 'Email',
-        dataIndex: 'email',
-        key: 'email',
-      },
       {
         title: 'Teams',
         dataIndex: 'teams',
         key: 'teams',
-        render: (teams: EntityReference[]) => getTeamsText(teams),
+        render: (teams: EntityReference[]) =>
+          getEntitiesText(teams, EntityType.TEAM),
```

Comment:
As we have a details page for both teams and roles so can we make them clickable?

Reply:
Like we are doing for roles and policy listing page.
![image](https://user-images.githubusercontent.com/59080942/188132174-8ffa6cd7-9963-4b76-bc2f-1aae3d38057d.png)

Reply:
Made the changes